### PR TITLE
Added error field in traceFailure for SettingsV2 Component

### PR DESCRIPTION
**Pull request by srinathnarayanan**

### CODE REVIEW

**Comment by swviswan** [comment]
>       databaseAccountName: this.container.databaseAccount()?.name,

need db and collection name here?

---
Refers to: src/Explorer/Controls/Settings/SettingsComponent.tsx:268 in 14dda17. [](commit_id = 14dda1733b490a6275a9abc5ae727f873bb27c1e, deletion_comment = False)

**Review by swviswan — APPROVED** [review]
:shipit:

**Comment by srinathnarayanan** [comment]
> > ```
> >   databaseAccountName: this.container.databaseAccount()?.name,
> > ```
> 
> need db and collection name here?
> 
> Refers to: src/Explorer/Controls/Settings/SettingsComponent.tsx:268 in [14dda17](https://github.com/Azure/cosmos-explorer/commit/14dda1733b490a6275a9abc5ae727f873bb27c1e). [](commit_id = [14dda17](https://github.com/Azure/cosmos-explorer/commit/14dda1733b490a6275a9abc5ae727f873bb27c1e), deletion_comment = False)

Added it to be consistent with other traceFailure logs

File: src/Explorer/Controls/Settings/SettingsComponent.tsx [hunk 1]
```diff
@@ -457,6 +457,8 @@ export class SettingsComponent extends React.Component<SettingsComponentProps, S
         Action.SettingsV2Updated,
         {
           databaseAccountName: this.container.databaseAccount()?.name,
+          databaseName: this.collection && this.collection.databaseId,
```
Comment (gitanuj) [inline src/Explorer/Controls/Settings/SettingsComponent.tsx:460]:
```suggestion
          databaseName: this.collection?.databaseId,
```

File: src/Explorer/Controls/Settings/SettingsComponent.tsx [hunk 2]
```diff
@@ -457,6 +457,8 @@ export class SettingsComponent extends React.Component<SettingsComponentProps, S
...
+          collectionName: this.collection && this.collection.id(),
```
Comment (gitanuj) [inline src/Explorer/Controls/Settings/SettingsComponent.tsx:461]:
```suggestion
          collectionName: this.collection?.id(),
```

File: src/Explorer/Controls/Settings/SettingsComponent.tsx [hunk 3]
```diff
@@ -471,9 +473,12 @@ export class SettingsComponent extends React.Component<SettingsComponentProps, S
         Action.SettingsV2Updated,
         {
           databaseAccountName: this.container.databaseAccount()?.name,
+          databaseName: this.collection && this.collection.databaseId,
```
Comment (gitanuj) [inline src/Explorer/Controls/Settings/SettingsComponent.tsx:476]:
```suggestion
          databaseName: this.collection?.databaseId,
```

File: src/Explorer/Controls/Settings/SettingsComponent.tsx [hunk 4]
```diff
@@ -471,9 +473,12 @@ export class SettingsComponent extends React.Component<SettingsComponentProps, S
...
+          collectionName: this.collection && this.collection.id(),
```
Comment (gitanuj) [inline src/Explorer/Controls/Settings/SettingsComponent.tsx:477]:
```suggestion
          collectionName: this.collection?.id(),
```

**Review by gitanuj — APPROVED** [review]

File: src/Explorer/Controls/Settings/SettingsComponent.tsx (hunk 3 above)
Comment (vignesh-mr) [inline src/Explorer/Controls/Settings/SettingsComponent.tsx:476]: this.collection?.databaseId ?

File: src/Explorer/Controls/Settings/SettingsComponent.tsx (hunk 4 above)
Comment (vignesh-mr) [inline src/Explorer/Controls/Settings/SettingsComponent.tsx:477]: this.collection?.id()

**Review by vignesh-mr — APPROVED** [review]

**Pull request merged**